python/tvm/relay/frontend/pytorch.py

```diff
@@ -3785,6 +3836,8 @@ def create_convert_map(self):
             "aten::nonzero": self.nonzero,
             "aten::nonzero_numpy": self.nonzero_numpy,
             "aten::scatter": self.scatter,
+            "aten::scatter_add": self.scatter_add,
```

Comment:
Looks like it's an artifact from [PR#14008](https://github.com/apache/tvm/pull/14008)?

Reply:
No. Here I reordered scatter-like operations in one sequence